com.unity.render-pipelines.high-definition-config/CHANGELOG.md

```diff
@@ -4,7 +4,12 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [11.0.0] - 2020-10-21
+## [2022.2.0] - 2021-11-1
```

Comment:
Shouldn't it be 2021.2.0 instead? 
Note: For all packages

Reply:
good catch!